.travis/rpmbuild_test

```diff
@@ -14,6 +14,7 @@ rpm -iv ~/rpmbuild/RPMS/*/*.rpm
 
 if [ "$OS_VERSION" = 6 ]; then
     # the test suite has not yet been ported to centos6 python, skip it
+    echo "the test suite has not yet been ported to centos6 python, skipping."
```

Comment:
Might as well just remove the comment above since you added this message don't you think?

Reply:
welcome to the Department of Redundancy department! Yes :)